Changelog.md

```diff
@@ -27,6 +27,7 @@ Compiler Features:
 Bugfixes:
  * Type Checker: Fix internal compiler error on tuple assignments with invalid left-hand side.
  * Yul IR Code Generation: Fix internal compiler error when accessing the ``.slot`` member of a mapping through a storage reference in inline assembly.
+ * Natspec: Fix event's Natspec inheritance for devdoc.
```

Comment:
Looking at the original issue, I don't think this is a bug, rather a missing feature.

Reply:
I changed the words and put it under features. How does it sound now?

Reply:
Yup, that looks better, at least to me, although I'm not sure whether NatSpec is a compiler or a language feature. @christianparpart will have to provide that input.

Reply:
> I'm not sure whether NatSpec is a compiler or a language feature.

In the changelog I see that it can be either but it's generally under "Language Features" when we touch the syntax and under "Compiler Features" when it's about the natspec output from the compiler. In this case I'd say it's the latter and there's actually one other inheritance related case that was classified like that.

> I don't think this is a bug, rather a missing feature.

You could also argue that it was a bug in #9338 because I don't see any good reason why it did not cover both. But I'm fine with classifying it as a missing feature too. It's kinda both.

BTW, that older PR did not even add a changelog entry.